ERROR - 2024-09-04 00:21:07 --> mysqli_sql_exception: Invalid default value for 'is_admin' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\food\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\food\app\Database\Migrations\2024-09-03-225918_CriaTabelaUsuarios.php(86): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(867): App\Database\Migrations\CriaTabelaUsuarios->up()
#6 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\food\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-04 18:59:18 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'food', 3306, '', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\food\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\food\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(697): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\food\system\Commands\Database\MigrateRollback.php(86): CodeIgniter\Database\MigrationRunner->getLastBatch()
#8 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\food\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\food\system\Database\MigrationRunner.php(697): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\food\system\Commands\Database\MigrateRollback.php(86): CodeIgniter\Database\MigrationRunner->getLastBatch()
#7 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#8 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#9 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-09-04 21:57:45 --> mysqli_sql_exception: Unknown column 'nome' in 'field list' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\food\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\food\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\food\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\food\app\Database\Seeds\UsuarioSeeder.php(19): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\food\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#9 C:\xampp\htdocs\food\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-09-04 22:11:48 --> mysqli_sql_exception: Unknown column 'nome' in 'field list' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\food\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\food\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\food\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\food\app\Database\Seeds\UsuarioSeeder.php(27): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\food\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#9 C:\xampp\htdocs\food\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-09-04 22:15:32 --> mysqli_sql_exception: Duplicate entry 'admin@admin.com' for key 'email' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\food\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\food\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\food\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\food\app\Database\Seeds\UsuarioSeeder.php(19): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\food\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#9 C:\xampp\htdocs\food\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-09-04 22:31:53 --> mysqli_sql_exception: Unknown column 'nome' in 'field list' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\food\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\food\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\food\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\food\app\Database\Seeds\UsuarioSeeder.php(19): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\food\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#9 C:\xampp\htdocs\food\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2024-09-04 22:36:29 --> mysqli_sql_exception: Duplicate entry 'admin@admin.com' for key 'email' in C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\food\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\food\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\food\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\food\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\food\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\food\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\food\app\Database\Seeds\UsuarioSeeder.php(19): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\food\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#9 C:\xampp\htdocs\food\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\food\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\food\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\food\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\food\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\food\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
